.gitignore

```diff
@@ -1,14 +1,149 @@
-build/*
-dist/*
-py/frequenz_api_dispatch.egg-info/*
-py/frequenz/api/dispatch/**/*_pb2.py
-py/frequenz/api/dispatch/**/*_pb2.pyi
-py/frequenz/api/dispatch/**/*_pb2_grpc.py
-py/frequenz/api/dispatch/**/*_pb2_grpc.pyi
```

Comment:
These should stay too. We might want to add a more general rule that ignore generated files in the repo-config template though. Feel free to create an issue (or PR!).